### ISSUE TITLE
No entities created

### ISSUE THREAD

**Issue by brgerig** [rytilahti/homeassistant-upnp-availability#2]
I just installed from HACS and added the integration. Running on core-2021.3.2 in a python virtual environment.

After adding the integration, it doesn't create any entities. I turned on debug logging, and the only thing I see for upnp_availability is:
```
2021-03-12 14:52:14 DEBUG (MainThread) [custom_components.upnp_availability.binary_sensor] Setting up upnp_availability..
2021-03-12 14:52:14 DEBUG (MainThread) [custom_components.upnp_availability.upnpstatustracker] Going to request upnp:rootdevices in the network..

```
I see lots of ssdp traffic while capturing packets on port 1900 on the box HA is running on, including ssdp:alive and ssdp:byebye notifications. I'm testing by pausing/unpausing both an xbox one and a Sony Bluray player.

**Comment by rytilahti** [comment]
I assume #1 is not related to this problem? This could very well be related to having multiple interfaces... Regarding to pausing/unpausing, this integration will not detect any such changes, but just whether the device has reported that it's on or off.

**Comment by brgerig** [comment]
Nope, #1 was solely a a problem with install from HACS because github downloads were broken.

I'm not in docker, I'm running core on my own Linux install straight from python. Only one network interface.

Just fixed it, though! Your comment in #1 about listening on interfaces finally made me think "Firewall." I stopped the firewall on my HA server, turned things on, and immediately stuff started popping up. I'll probably remove it since what I really need is the play/pause status, but I did at least want to get it working first!

So, for future reference: check your firewall, folks.
